vision/imageloader/imageloader.go

```diff
@@ -91,34 +91,36 @@ func (p *ImageLoader) read() {
 			p.errChan <- err
 			break
 		}
-		p.samples <- sample{p.trans1.Run(m).(image.Image), label}
+		inputs = append(inputs, p.trans.Run(m).(torch.Tensor))
+		labels = append(labels, int64(label))
+		if len(inputs) == p.mbSize {
+			d := torch.Stack(inputs, 0)
+			l := torch.NewTensor(labels)
+			runtime.KeepAlive(&d)
```

Comment:
We don't have to `KeepAlive` `d` and `l` here because they use used later and will not be freed so soon.

Reply:
done